css/style.css

```diff
@@ -1,11 +1,227 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+header {
+  display: flex;
+  padding: 40px;
+  margin-left: 100px;
+}
+
+/* Main Image*/
+.main_image {
+  margin-bottom: 600px;
+  position: relative;
+  top: -70px;
```

Comment:
You should need this here. This section should just sit under the head naturally - no need to force it up or down with the `top` property